kfdefs/overlays/moc/zero/opf-monitoring/scrape-configs/additional-scrape-configs.yaml

```diff
@@ -19,3 +19,50 @@ stringData:
       - targets: ["argocd-repo-server-metrics-argocd.apps.moc-infra.massopen.cloud"]
         labels:
           service: argocd-repo-server
+
+    # Metrics for thoth-station services
+    - job_name: 'pushgateway'
```

Comment:
@harshad16 I think you can add all the targets as a single job since their configuration is the same.

Reply:
@4n4nd can we have it in differnt job if it alright?

Reply:
Sure 👍
Can you prefix all job names with "Thoth"?

Reply:
yes, we have decided to move to single job configuration and using labels.